recipes/spyder/meta.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: spyder
+  version: 2.3.9
+
+source:
+  fn:  spyder-2.3.9.zip
+  url: https://pypi.python.org/packages/fb/37/09b789dbe321894afe1657f11f0026a5ea79987f1c30176b92ee648b3faa/spyder-2.3.9.zip
+  md5: dd01e07a77123c128ff79ba57b97c1d7
+
+build:
+  number: 0
+  osx_is_app: True
```

Comment:
Did not know about this. Maybe @conda-forge/xonsh should be using something like this.